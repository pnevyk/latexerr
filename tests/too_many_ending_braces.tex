\documentclass{article}

\begin{document}
\date April 2018}
\end{document}
