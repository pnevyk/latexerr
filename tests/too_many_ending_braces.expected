File: too_many_ending_braces.tex

Error on line 4: Number of curly braces near \date April 2018} does not match.
